scico/loss.py

```diff
@@ -51,27 +51,34 @@ def __init__(
         self,
         y: Union[JaxArray, BlockArray],
         A: Optional[Union[Callable, operator.Operator]] = None,
-        scale: float = 0.5,
+        l: Optional[functional.Functional] = None,
```

Comment:
I'm on the fence about naming it l. (Yes that is an l and not an I. Also it is an `l` and not a `1`). See the problem?

Reply:
This occurred to me too, but all the other parameter names set the pattern of matching the labels used in the associated equation. Any suggestions for alternatives, which would probably also need to be used in the equation?

Reply:
Perhaps we want to make it `L` for now and then this issue can be grouped in a combined new PR `Ambiguity in symbol L as Lipschitz constant or loss`. (See other comment below)

Reply:
Change it to `f`.